deployment/service.js

```diff
@@ -0,0 +1,15 @@
+const Service = {
+	type: 'object',
+	additionalProperties: false,
+	properties: {
+		port: {
+			type: 'number',
+			minimum: 1,
+			maximum: 65535
```

Comment:
actually we probably shouldn't allow ephemeral ports?

Reply:
Ya that's a good point. You wouldn't know what to define it as here 😄

Reply:
32767 should be the last port, ports after that are allocated as ephemeral ports by IANA and it seems we are doing so too.